airflow/contrib/hooks/qubole_hook.py

```diff
@@ -182,6 +185,29 @@ def get_jobs_id(self, ti):
             cmd_id = ti.xcom_pull(key="qbol_cmd_id", task_ids=self.task_id)
         Command.get_jobs_id(self.cls, cmd_id)
 
+    def get_redirect_url(self, task, dttm):
+        session = settings.Session()
+        url = ''
+
+        try:
+            conn = BaseHook.get_connection(task.kwargs['qubole_conn_id'])
+            if conn and conn.host:
+                host = re.sub(r'api$', 'v2/analyze?command_id=', conn.host)
```

Comment:
is v2/analyze?command_id= generic used for qubole hook?

Reply:
yes